content/applications/sales/crm/track_leads/lead_scoring.rst

```diff
@@ -1,73 +1,163 @@
-=============================
-Assign leads based on scoring
-=============================
+=========================================
+Assign leads with predictive lead scoring
+=========================================
 
-*Lead Scoring* lets you rank and prioritize leads based on selected criteria. A higher score
-indicates that a lead is more likely to bring value to your company.
+The Odoo *CRM* app can automatically assign leads/opportunities to sales teams and salespeople. A
```

Comment:
Lets italicize the whole _Odoo CRM_

Reply:
She had it right the first time (asked ahead of time.)
I know it's confusing 😅 

Italicize just the app name please, as that's the product name.
When we're referring to the app itself, we italicize it. When we're generally talking about the noun, it does not get italicized.

Difference:
Odoo _Helpdesk_ vs. help desk
Odoo _Inventory_ vs. "...inventory valuation"

In the case of CRM, it's a little strange. You have to decide based on context (are we talking about the Odoo product, a navigational step, etc.).